File: java/Shirt.java
class Shirt extends Clothing
{
    private String color;
    private String sleeveType;

    public Shirt(int idProduct, String name, String brand, String price, String size, String material, String gender, String color, String sleeveType)
    {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public String toTableRow() 
    {
        return String.format("| %-2d | %-15s | %-10s | %-12s | %-4s | %-12s | %-6s | %-6s | %-11s |", idProduct, name, brand, price, size, material, gender, color, sleeveType);
    }
}

File: py/shirt.py
from clothing import Clothing

class Shirt(Clothing):
    def __init__(self, idProduct, name, brand, price, size, material, gender, color, sleeveType):
        super().__init__(idProduct, name, brand, price, size, material, gender)
        self.color = color
        self.sleeveType = sleeveType

    def toTableRow(self):
        return f"| {self.idProduct:<2} | {self.name:<12} | {self.brand:<8} | {self.price:<10} | {self.size:<4} | {self.material:<10} | {self.gender:<6} | {self.color:<6} | {self.sleeveType:<11} |"
